### PR TITLE
JBPM-7355 - Missing 'import' doesn't always produce a compilation err…

### PR DESCRIPTION
**Pull request by mswiderski**
…or in BPMN2 process

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/ImportClassTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public ImportClassTest() {
+        super(false);
+    }
+
+    @Test
+    public void testResourceType() throws Exception {
+        
+        assertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> { 
+            createKnowledgeBaseWithoutDumper("build/sample.bpmn", "build/sample2.bpmn"); })
+        .withMessageContaining("Process Compilation error HelloService cannot be resolved to a type");
```
Comment (MarianMacik) [inline jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/ImportClassTest.java]: Can we maybe check that it comes from sample2 process? If not, we can leave it as is.

**Pull request merged**